spring-cloud-dataflow-docs/src/main/asciidoc/architecture.adoc

```diff
@@ -56,19 +54,53 @@ The Data Flow server is also responsible for:
+
+* https://github.com/spring-cloud/spring-cloud-stream-binder-rabbit[RabbitMQ]
+* https://github.com/spring-cloud/spring-cloud-stream-binder-kafka[Apache Kafka]
+* https://github.com/spring-cloud/spring-cloud-stream-binder-aws-kinesis[Amazon Kinesis]
+* https://github.com/spring-cloud/spring-cloud-gcp/tree/master/spring-cloud-gcp-pubsub-stream-binder[Google PubSub (partner maintained)]
+* https://github.com/SolaceProducts/spring-cloud-stream-binder-solace[Solace PubSub+ (partner maintained)]
+* https://github.com/Microsoft/spring-cloud-azure/tree/master/spring-cloud-azure-eventhub-stream-binder[Azure Event Hubs (partner maintained)]
+
+<<applications,Pre-built Stream applications>> are available for RabbitMQ and Apache Kafka.
+You can easily modify these pre-built application using a version of https://start-scs.cfapps.io/[Spring Intializr for Stream applications].
+The Data Flow server is responsible for creating the destinations/topics that correspond to each pipe symbol and configure each application to produce or consume from the topics so the desired flow of data is achieved.
+
+The interaction of the main components for the three types of DSLs are shown in the following images.
+
+.The Spring Cloud Data Flow Stream DSL Architecure
+image::{dataflow-asciidoc}/images/arch-stream-dsl.png[The Spring Cloud Data Flow Streaming Application Architecture, scaledwidth="60%"]
+
+In this architecture, the Stream DSL is POSTed to the Data Flow Server.
+Based on the mapping of DSL application names to Maven and Docker artifacts, the http-source and cassandra-sink applications are deployed on the target runtime.
+Data that is posted to the HTTP application will then be stored in Cassandra.
+The https://docs.spring.io/spring-cloud-dataflow-samples/docs/current/reference/html/_streaming.html#spring-cloud-data-flow-samples-http-cassandra-overview[Samples Repository] shows this use case in full detail.
+
+The architecture for the Application DSL is similar.
+
+
+.The Spring Cloud Data Flow Application DSL Architecure
+image::{dataflow-asciidoc}/images/arch-app-dsl.png[The Spring Cloud Data Flow Application DSL Architecture, scaledwidth="60%"]
+
+In this architecture, the Application DSL description that describes a group of related applications is POSTed to the Data Flow Server.
```

Comment:
Maybe remove "description" from "the Application DSL description that describes"